packages/errors/src/messages.ts

```diff
@@ -23,6 +25,10 @@ export const SolanaErrorMessages: Readonly<{
 }> = {
     [SOLANA_ERROR__BLOCK_HEIGHT_EXCEEDED]:
         'The network has progressed past the last block for which this transaction could have been committed.',
+    [SOLANA_ERROR__EXPECTED_INSTRUCTION_TO_HAVE_ACCOUNTS]:
+        'The instruction with program address `$programAddress` and data `$data` does not have any accounts.',
+    [SOLANA_ERROR__EXPECTED_INSTRUCTION_TO_HAVE_DATA]:
+        'The instruction with program address `$programAddress` and accounts `$accountAddresses` does not have any data.',
```

Comment:
Since we don't know for sure that an account without data has accounts and vice versa, I wouldn't explicitly add them in the error message. I think it's fine to have in the context if someone wants to check their values but would be confusing in the error message for instruction that have neither.

Reply:
Ah that's what I was missing, I thought context was just about getting values into the error message but it's displayed separately too. Agreed, I'll remove the accounts/data from the messages and just include them in context.